spec/support/feature_spec_helper.rb

```diff
@@ -0,0 +1,89 @@
+module FeatureSpecHelper
+
+  RSpec.configure do |config|
```

Comment:
Consider moving the `RSpec.configure` block to below the module, *after* the final `end`